packages/near-api-js/src/providers/provider.ts

```diff
@@ -315,6 +315,7 @@ export interface AccessKeyWithPublicKey {
 export interface QueryResponseKind {
     block_height: BlockHeight;
     block_hash: BlockHash;
+    error: Error;
```

Comment:
Not totally sure about this one but it looks like the fields we reference directly at least overlap with the shape on the RPC error response.